ansible_collections/arista/avd/docs/release-notes/4.x.x.md

```diff
@@ -10,6 +10,16 @@
 
 ## Release 4.4.0
 
+### Deprecation Warnings for Python 3.8 and ansible-core<2.14
+
+- As per https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html#ansible-core-support-matrix, `ansible-core` 2.13 is EOL on November 6th 2023.
+
+  AVD will drop support for `ansible-core<2.14` in the next Minor Release after this date.
+
+- Because `ansible-core>=2.14` does not support Python < 3.9, the support for Python 3.8 will be dropped at the same time.
```

Comment:
```suggestion
- Since `ansible-core` versions above 2.14 do not support Python versions below 3.9, the support for Python 3.8 will be dropped from AVD at the same time.
```